examples/server/server.cpp

```diff
@@ -512,7 +515,7 @@ int main(int argc, char ** argv) {
 
             wparams.strategy = params.beam_size > 1 ? WHISPER_SAMPLING_BEAM_SEARCH : WHISPER_SAMPLING_GREEDY;
 
-            wparams.print_realtime   = false;
+            wparams.print_realtime   = params.print_realtime;
```

Comment:
```suggestion
            wparams.print_realtime   = false;
```
use `params.print_realtime` inside of `whisper_print_segment_callback` to suppress the print instead.

Reply:
Good catch, resolved in https://github.com/ggerganov/whisper.cpp/pull/1541/commits/0d4b5da60ec1b8c42e3ce9a4946a81110c9578b0.